timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -356,6 +390,30 @@ services:
         tags:
           - server-request-context
         returns: GetCommitTimestampResponse
```

Comment:
Should this endpoint (and therefore request/response pair) be like acquireNamedTimestampLeases?

The min part is really only for the getter no?

Reply:
fixing